### PR TITLE
fixes https://github.com/wso2/docs-apim/issues/4774

### PR DESCRIPTION
**Pull request by Mariangela**
## Purpose
fixes https://github.com/wso2/docs-apim/issues/4774
- PR sync for https://github.com/wso2/docs-apim/pull/4779

### CODE REVIEW

File: en/docs/deploy-and-publish/deploy-on-gateway/deploy-api/deploy-through-multiple-api-gateways.md [hunk 1]
```diff
@@ -2,138 +2,132 @@
...
+show_as_token_endpoint_url = true
+service_url = "https://localhost:9445/services/"
+username= "admin"
+password= "admin"
+http_endpoint = "http://localhost:8282"
+https_endpoint = "https://localhost:8245"
+```
+
+!!! tip
+        If you have multiple Gateways that support one type of key (e.g., when there are two Gateways that support the production keys, as seen in the above code snippet.), the environments you add via the `<API-M_HOME>/repository/conf/deployment.toml` file will be visible in a drop-down list of the API **Try Out** tab, which is in the Developer Portal of instance 1. This allows subscribers to send API requests to any selected Gateway.
```
Comment (Mariangela) [inline en/docs/deploy-and-publish/deploy-on-gateway/deploy-api/deploy-through-multiple-api-gateways.md:60]:
```suggestion
        If you have multiple Gateways that support one type of key (e.g., when there are two Gateways that support the production keys, as seen in the above code snippet), the environments you add via the `<API-M_HOME>/repository/conf/deployment.toml` file will be visible in a drop-down list of the API **Try Out** tab, which is in the Developer Portal of instance 1. This allows subscribers to send API requests to any selected Gateway.
```

File: en/docs/deploy-and-publish/deploy-on-gateway/deploy-api/deploy-through-multiple-api-gateways.md [hunk 2]
```diff
@@ -2,138 +2,132 @@
...
+2.  Click **Deployments** and click **Deploy New Revision**.
 
     <a href="{{base_path}}/assets/img/learn/api-revisions-mutiple-gateways.png"><img src="{{base_path}}/assets/img/learn/api-revisions-mutiple-gateways.png" title="Api Revisions Multiple Gateways" width="80%" alt="Api Revisions Multiple Gateways"/></a>
     
     Note that the two Gateway environments are listed there
 
     <a href="{{base_path}}/assets/img/learn/multiple-gateways-revision-dialog.png"><img src="{{base_path}}/assets/img/learn/multiple-gateways-revision-dialog.png" title="Multiple Gateways Dialog" width="40%" alt="Multiple Gateways Dialog"/></a>
 
-9.  Select both gateways and click deploy.
+3.  Select both gateways and click deploy.
```
Comment (Mariangela) [inline en/docs/deploy-and-publish/deploy-on-gateway/deploy-api/deploy-through-multiple-api-gateways.md:126]:
```suggestion
3.  Select both Gateways and click deploy.
```

File: en/docs/deploy-and-publish/deploy-on-gateway/deploy-api/deploy-through-multiple-api-gateways.md [hunk 3]
```diff
@@ -2,138 +2,132 @@
...
 
-7.  Sign in to the API Publisher in the **first** WSO2 API-M instance and click to edit an API.
+1.  Sign in to the API Publisher in the **first** WSO2 API-M instance and click to edit an API.
 
     <a href="{{base_path}}/assets/img/learn/pizzashak-api-publisher.png"><img src="{{base_path}}/assets/img/learn/pizzashak-api-publisher.png" title="Pizza Shack Publisher" width="30%" alt="Pizza Shack Publisher"/></a>
      
-8.  Click **Deployments** and click **Deploy New Revision**.
+2.  Click **Deployments** and click **Deploy New Revision**.
 
     <a href="{{base_path}}/assets/img/learn/api-revisions-mutiple-gateways.png"><img src="{{base_path}}/assets/img/learn/api-revisions-mutiple-gateways.png" title="Api Revisions Multiple Gateways" width="80%" alt="Api Revisions Multiple Gateways"/></a>
```
Comment (Mariangela) [inline en/docs/deploy-and-publish/deploy-on-gateway/deploy-api/deploy-through-multiple-api-gateways.md:120]:
```suggestion
    <a href="{{base_path}}/assets/img/learn/api-revisions-mutiple-gateways.png"><img src="{{base_path}}/assets/img/learn/api-revisions-mutiple-gateways.png" title="API Revisions Multiple Gateways" width="80%" alt="API Revisions Multiple Gateways"/></a>
```

**Comment by Mariangela** [comment]
@samgnaniah as this is a PR sync and as Tharindu approved the main PR. Please go ahead and approve and merge this pls.

**Review by samgnaniah — APPROVED** [review]

**Pull request merged**